src/api.coffee

```diff
@@ -71,7 +71,7 @@ apiHelper = (Actions, listenAction, successAction, httpMethod, pathMaker) ->
         statusCode = jqXhr.status
         if statusCode is 200
           # HACK For PUT returning nothing (actually, it returns HTML for some reason)
-          successAction(args...)
+          successAction('', args...)
```

Comment:
i checked, but i can check again.  this should only be happening when we are faking PUTs and PATCHes on our end, and the correct/expected behavior is that ```args...``` is the second argument to follow the pattern set throughout the api helper as in line 68.